ui/app/mainui/AppMain.qml

```diff
@@ -3,22 +3,22 @@ import QtQuick.Controls 2.13
 import QtQuick.Layouts 1.13
 import QtMultimedia 5.13
 import Qt.labs.qmlmodels 1.0
+import Qt.labs.platform 1.1
+import Qt.labs.settings 1.0
+
+import applayouts.Wallet 1.0
```

Comment:
It seems changing the imports from lowercase `applayouts` to camel-case `AppLayouts` will fix it

Reply:
I now see the comment that it should be the other way around.

Reply:
PR updated to change the imports